README.md

```diff
@@ -128,13 +128,30 @@ go install github.com/charmbracelet/vhs@latest
 
 ## The VHS Server
 
-VHS has an SSH server built in! When you self host VHS you can access it as
+VHS has an SSH server built in! When you self-host VHS you can access it as
```

Comment:
In this case "self host" actually doesn't need a hyphen because it's not modifying a noun.